.github/workflows/cd-sql-engine-tests.yaml

```diff
@@ -15,7 +15,7 @@ env:
 jobs:
   snowflake-tests:
     environment: DW_INTEGRATION_TESTS
-    if: ${{ github.event.action != 'labeled' || github.event.label.name == 'Run Tests With Other SQL Engines' }}
```

Comment:
Removing the first part of this check breaks all other triggers, because it requires the event to have this label name.